builder/amazon/common/step_security_group.go

```diff
@@ -45,7 +45,9 @@ func (s *StepSecurityGroup) Run(state multistep.StateBag) multistep.StepAction {
 
 	port := s.CommConfig.Port()
 	if port == 0 {
-		panic("port must be set to a non-zero value.")
+		if s.CommConfig.Type != "none" {
+			panic("port must be set to a non-zero value.")
```

Comment:
I feel like removing support for it (at least for the AWS builder) is also a viable option, but I don't have a good sense for how people are using it